.github/bin/minikube/test-backup-restore.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+set -e
+set -x
+
+# Get absolute path for root repo directory from github actions context: https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions
+export OPERATOR_REPO="${GITHUB_WORKSPACE}"
+if [ -z "${OPERATOR_REPO}" ]; then
+  SCRIPT=$(readlink -f "${BASH_SOURCE[0]}")
+  OPERATOR_REPO=$(dirname "$(dirname "$(dirname "$(dirname "$SCRIPT")")")")
+fi
+source "${OPERATOR_REPO}"/.github/bin/common.sh
+source "${OPERATOR_REPO}/olm/olm.sh"
+
+# Stop execution on any error
+trap "catchFinish" EXIT SIGINT
+
+createBackupCR() {
+  kubectl apply -f - <<EOF
+apiVersion: org.eclipse.che/v1
+kind: CheClusterBackup
+metadata:
+  name: eclipse-che-backup
+  namespace: ${NAMESPACE}
+spec:
+  triggerNow: true
+  autoconfigureRestBackupServer: true
```

Comment:
Is it backup server under `servers[rest]` or `servers[internal]`?

Reply:
`autoconfigureRestBackupServer` configures `internal` server that has the same type as `rest` one.